README.md

```diff
@@ -32,13 +36,19 @@ To run the code in this project, first create a Python virtual environment using
 conda create -n handbook python=3.10 && conda activate handbook
 ```
 
-Next, install PyTorch v2.1.0. Since this hardware-dependent, we
+Next, install PyTorch `v2.1.0` - the precise version is important for reproducibility! Since this is hardware-dependent, we
 direct you to the [PyTorch Installation Page](https://pytorch.org/get-started/locally/).
 
-Once PyTorch is installed, you can install the remaining package dependencies as follows:
+You can then install the remaining package dependencies as follows:
+
+```shell
+python -m pip install .
+```
+
+You will also need Flash Attention 2 installed, which can be done by running:
 
 ```shell
-pip install .
+python -m pip install flash-attn --no-build-isolation
```

Comment:
Do you think we should add 
```bash
pip install ninja packaging
```
since otherwise installation will fail.

Reply:
I think this is included by default in `torch`, but sure let's add it!

Reply:
ah it's already included by `pip install .` in the previous step